js/modules/Debug/Debug.js

```diff
@@ -382,15 +384,35 @@ window.GLPI.Debug = new class Debug {
     }
 
     refreshWidgetButtons() {
-        const server_performance_button = this.getWidgetButton('server_performance');
+        // Server performance
         const server_perf = this.initial_request.server_performance;
         const memory_usage_mio = (server_perf.memory_usage / 1024 / 1024).toFixed(2);
-        server_performance_button.find('.debug-text').text(`${server_perf.execution_time}ms | ${memory_usage_mio}mio`);
-
-        const ajax_requests_button = this.getWidgetButton('ajax_requests');
-        ajax_requests_button.find('.debug-text').text(this.ajax_requests.length);
-
-        this.getWidgetButton('requests').find('.debug-text').text('Requests');
+        const server_performance_button_label = __("$1s ms using $2s mio")
```

Comment:
See my other comment on translations. Also, if we keep translations on debug bar, this one should use the same domain as all other.